lib/vagrant-lxc/plugin.rb

```diff
@@ -8,12 +8,16 @@ class Plugin < Vagrant.plugin("2")
       The LXC provider allows Vagrant to manage and control
       LXC-based virtual machines.
       EOF
+      locale_loaded = false
 
       provider(:lxc, parallel: true, priority: 7) do
```

Comment:
This appears to be in a parallel section, do we need to do some sort of locking here?